### PR TITLE
📐Update TimeEntry cell

### PR DESCRIPTION
**Pull request by sofokli1**
I used the cell that was there already. It is may need some tweaks but it is good enough for a testflight
🦑

### CODE REVIEW

File: TogglTrack/Utils/TimeIntervalExtensions.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@ public extension TimeInterval
         let minutes = (time / 60) % 60
         let hours = (time / 3600)
 
-        return String(format: "%0.2d:%0.2d:%0.2d.%0.3d",hours,minutes,seconds,ms)
+        return String(format: "%0.2d:%0.2d:%0.2d",hours,minutes,seconds,ms)
```
Comment (Odrakir) [inline TogglTrack/Utils/TimeIntervalExtensions.swift]: I think we should remove the first leading zero as we do in the improved format in the mobile app.

File: TogglWatch WatchKit Extension/Views/TimeEntryCellView.swift [hunk 2]
```diff
@@ -20,29 +21,46 @@ public struct TimeEntryCellView: View
...
+            HStack {
+                Spacer()
+                Text(viewModel.durationString)
+                    .font(.system(size: 14))
+                    .foregroundColor(Color.init(red: 174/255, green: 180/255, blue: 191/255))
+                    .multilineTextAlignment(.trailing)
+            }
         }
+        .padding()
+        .background(Color.init(red: 242/255, green: 244/255, blue: 252/255, opacity: 0.07))
```
Comment (Odrakir) [inline TogglWatch WatchKit Extension/Views/TimeEntryCellView.swift]: This results in a weird double background. I think what you are looking for is `listRowPlatterColor` (which I don't know if it should go here or in the List)

File: TogglTrack/Extensions/ColorExtension.swift [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+//
+
+import SwiftUI
+
+extension Color
+{
+    public init?(hex: String) {
+        let r, g, b, a: Double
+
+        if hex.hasPrefix("#") {
```
Comment (Odrakir) [inline TogglTrack/Extensions/ColorExtension.swift]: Early exit, please.

File: TogglTrack/Extensions/ColorExtension.swift [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            let start = hex.index(hex.startIndex, offsetBy: 1)
+            let hexColor = String(hex[start...])
+
+            if hexColor.count == 8 {
```
Comment (Odrakir) [inline TogglTrack/Extensions/ColorExtension.swift]: I think our colors are 6 characters long, so probably this whole method doesn't work. Take a look at the Lite's code, I think we did something similar.

File: TogglTrack/Utils/TimeIntervalExtensions.swift (hunk 1 above)
Comment (sofokli1) [reply TogglTrack/Utils/TimeIntervalExtensions.swift]: I do not mind dropping, I was just following zeplin

File: TogglTrack/Extensions/ColorExtension.swift (hunk 3 above)
Comment (sofokli1) [reply TogglTrack/Extensions/ColorExtension.swift]: this will be copied from lite

**Pull request merged**